README_sqlite_dbs.md

```diff
@@ -7,12 +7,27 @@ This directory contains SQLite databases that make our clean tabular
 There are DBs for the versions of the MCRF OMOP CDM data, the CDM
 vocabulary, and examples for various classification tasks.
 
+### naming convention
+
+The emr DBs are named following the convention 
+'db_name.data_date.tags.sqlite3' where db_name is
+one of 'mcrf', 'emr', 'vocab', data_date (YYYYMMDD) indicates the version of 
+MCRF data based on d/l date, and tags is an extensible dotted field 
+(ex ...data_date.tag.tag.sqlite3) that can be used to provide addional
```

Comment:
I would vote for the syntax being `<tag> ("-" <tag>)+` so that splitting by dots still recovers the semantic chunks of the filename.

additional